topics/dev/tutorials/conda_sys/slides.html

```diff
@@ -10,9 +10,9 @@
 ### Environment variables
 
 ```sh
-$ MY_NAME="Anthony"
+$ MY_NAME="Bobby"
 $ echo $MY_NAME
-Anthony
+Bobby
```

Comment:
New name? ;)

Reply:
Yeah, this was getting too personal! (and bobby is so beautiful)